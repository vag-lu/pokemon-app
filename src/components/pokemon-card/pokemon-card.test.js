import { shallow, configure } from 'enzyme';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import PokemonCard from '.';

configure({ adapter: new Adapter() });

const bulbasaur = {
    name: "bulbasaur",
    id: 1,
    sprites: {
        front_default: "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png"
    },
    types: [
        {
          slot: 1,
         type: {
            name: "grass",
            url: "https://pokeapi.co/api/v2/type/12/"
          }
        },
        {
          slot: 2,
          type: {
            name: "poison",
            url: "https://pokeapi.co/api/v2/type/4/"
          }
        }
      ],

}

describe('PokemonCard', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<PokemonCard item={bulbasaur}/>);
        expect(wrapper).toMatchSnapshot();
    })
})