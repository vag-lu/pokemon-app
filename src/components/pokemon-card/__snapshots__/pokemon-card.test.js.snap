// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonCard renders correctly 1`] = `ShallowWrapper {}`;
